packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -1166,60 +1172,59 @@ function handleErrorMsg(err) {
 }
 
 /**
- * create an importable CSV file, to utilize with the migration script.
+ * Generate a CSV file that can be imported for use with the migration script.
  * @param {*} payload api request payload
  * @param {*} taxonomies taxonomies data
  * @returns
  */
 async function createImportableCSV(payload, taxonomies) {
-  let taxonomiesData = [];
-  let headers = ['Taxonomy Name', 'Taxonomy UID', 'Taxonomy Description'];
-  for (let index = 0; index < taxonomies?.length; index++) {
-    const taxonomy = taxonomies[index];
-    const taxonomyUID = taxonomy?.uid;
-    if (taxonomyUID) {
-      const sanitizedTaxonomy = sanitizeData({
-        'Taxonomy Name': taxonomy?.name,
-        'Taxonomy UID': taxonomyUID,
-        'Taxonomy Description': taxonomy?.description,
-      });
-      taxonomiesData.push(sanitizedTaxonomy);
-      payload['taxonomyUID'] = taxonomyUID;
-      const terms = await getAllTermsOfTaxonomy(payload);
-      //fetch all parent terms
-      const parentTerms = terms.filter((term) => term?.parent_uid === null);
-      const termsData = getParentAndChildTerms(parentTerms, terms, headers);
-      taxonomiesData.push(...termsData);
+  try {
+    let taxonomiesData = [];
+    let headers = [];
+    for (let index = 0; index < taxonomies?.length; index++) {
+      const taxonomy = taxonomies[index];
+      const taxonomyUID = taxonomy?.uid;
+      if (taxonomyUID) {
+        payload['type'] = 'export-taxonomies';
+        payload['format'] = 'csv';
+        payload['taxonomyUID'] = taxonomyUID;
+        const data = await taxonomySDKHandler(payload);
+        const taxonomies = await csvParse(data, headers);
```

Comment:
[MINOR] This variable allocation maybe unnecessary. Not sure if it was kept for better code readability.

Reply:
@antonyagustine , replaced for loop with for ... of loop. Removed taxonomy and repeated payload variable.Using data & taxonomies for better code readability.